### PR TITLE
#19766 Fixes incorrect limit

### LINKED ISSUE #19766: Incorrect limit
`limit(2**(-x)*sqrt(4**(x+1)+1),x,oo)` gives `oo`, when in fact it is `2`.
I tried, to simplify expression and move `2**(-x)` under the `sqrt`, but couldn't. What function performs this?

### PR DESCRIPTION
**Pull request by shivangdubey**
Fixes partially #19766 
Added a test case for the limit.
`limit(2**(-x)*sqrt(4**(x+1)+1), x, oo)`

NO ENTRY

### CODE REVIEW

File: sympy/series/tests/test_limits.py [hunk 1]
```diff
@@ -861,3 +861,6 @@ def test_issue_15055():
 
 def test_issue_19739():
     assert limit((-S(1)/4)**x, x, oo) == 0
+
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:864]: Maintain two line spacing.

File: sympy/series/tests/test_limits.py [hunk 2]
```diff
@@ -861,3 +861,6 @@ def test_issue_15055():
...
+def test_issue_19766():
+    assert limit(2**(-x)*sqrt(4**(x+1)+1), x, oo) == 2
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:866]: Add space around `+` signs.

**Comment by sachin-4099** [comment]
Add `NO ENTRY` in release notes,

**Comment by sachin-4099** [comment]
Just squash both the commits into one and add a more suitable `commit` message.
The travis-build will fail due to this: https://github.com/sympy/sympy/issues/19768

When this issue gets resolved we will restart the build and merge this PR.

**Comment by shivangdubey** [comment]
> Just squash both the commits into one and add a more suitable `commit` message.
> 
> The travis-build will fail due to this: #19768
> When this issue gets resolved we will restart the build and merge this PR.

Alright! The commits have been squashed and pushed.

File: sympy/core/tests/test_evalf.py [hunk 3]
```diff
@@ -579,3 +579,7 @@ def test_issue_13425():
 
 def test_issue_17421():
     assert N(acos(-I + acosh(cosh(cosh(1) + I)))) == 1.0*I
+
+
+def test_issue_19774():
+    assert exp(x/2).evalf() == exp(0.5*x)
```
Comment (sachin-4099) [inline sympy/core/tests/test_evalf.py:585]: Why was this testcase added? Testcase should be added only for those cases which are working.

Comment (shivangdubey) [reply sympy/core/tests/test_evalf.py]: #19774

**Comment by sachin-4099** [comment]
I have restarted the build.
@shivangdubey please mention:
`Partially Fixes #19766` instead of `Fixes #19766`.

**Comment by sachin-4099** [comment]
Looks good to me. This can be merged @jksuom.

**Comment by jksuom** [comment]
Thanks.

**Pull request merged**